jenkins/pipeline.groovy

```diff
@@ -240,6 +244,15 @@ pipeline {
                                     sleep 5
                                 done
 
+                                if [[ \$CI_FS_MTR == 'yes' ]]; then
+                                    if [[ ! -f /mnt/ci_disk_\$CMAKE_BUILD_TYPE.img ]] && [[ -z \$(mount | grep /mnt/ci_disk_dir_\$CMAKE_BUILD_TYPE) ]]; then
+                                        sudo dd if=/dev/zero of=/mnt/ci_disk_\$CMAKE_BUILD_TYPE.img bs=1G count=10
+                                        sudo /sbin/mkfs.vfat /mnt/ci_disk_\$CMAKE_BUILD_TYPE.img
+                                        sudo mkdir -p /mnt/ci_disk_dir_\$CMAKE_BUILD_TYPE
+                                        sudo mount -o loop -o uid=27 -o gid=27 -o check=r /mnt/ci_disk_\$CMAKE_BUILD_TYPE.img /mnt/ci_disk_dir_\$CMAKE_BUILD_TYPE
```

Comment:
Magic numbers `27`
May be
```
-o uid=`id -u ...` -o gid=`id - g ...`
```

Reply:
Those IDs are from docker image:
```
uid=27(mysql) gid=27(mysql) groups=27(mysql)
```
If we use `id -u` it'll return id of the worker user

Reply:
And this ID/GID is hardcoded here: https://github.com/Percona-Lab/ps-build/blob/8.0/docker/Dockerfile.inc#L9

Reply:
To be exact, it'll return:
```
uid=1000(ec2-user) gid=1000(ec2-user)
```

Reply:
Actually, I meant
-o uid=`id -u mysql` -o gid=`id - g mysql`
But I think I understand that it's not possible as there is no `mysql` user outside docker image (in the worker OS).
So, hardcoded values are fine then.

Reply:
It's not possible, as this user don't exist on worker node, only inside ps-build image